src/module.ts

```diff
@@ -50,6 +50,11 @@ export interface SanityModuleVisualEditingOptions {
    * @default 'live-visual-editing'
    */
   mode?: SanityVisualEditingMode
+  /**
+   * Set proxy endpoint for fetching preview data
+   * @default '/_sanity/fetch'
+   */
+  proxyEndpoint?: string
```

Comment:
why do we need a proxy endpoint?

Reply:
I hope I'm not fundamentally missing how Nuxt handles data fetching here:

- The initial request is performed on the server where the Sanity client has a token to fetch preview data.
- On subsequent page navigations, when in preview mode, we want to fetch more preview data.
- We can't do that on the client as we don't have access to the token.
- Instead we just send the same params to this endpoint, which verifies the preview cookie and performs a fetch using a client with access to the token.

Maybe there's a simpler way to do this though!